modules/central-services-public-config.adoc

```diff
@@ -96,8 +96,9 @@ include::snippets/technology-preview.adoc[]
 |`Central.declarativeConfiguration.mounts.secrets`
 | Mounts secrets used for declarative configurations.
 
-| `central.disableTelemetry`
-| Use `true` to disable online telemetry data collection.
+//ROX-13807
+//| `central.disableTelemetry`
+//| Use `true` to disable online telemetry data collection.
 //TODO: Add link to telemetry
```

Comment:
```suggestion
```

Why not delete those lines instead of commenting them out?

Reply:
Sure, we can do that.